www/nginx/src/opnsense/mvc/app/views/OPNsense/Nginx/index.volt

```diff
@@ -85,10 +171,38 @@
     #frm_ipacl_dlg .bootstrap-select {
         width: 100% !important;
     }
-    .filter-option {
-        padding: inherit !important;
+    #nginx_conf_container {
```

Comment:
since this becomes quite large, it may make sense to extract it to its own file.